### ISSUE TITLE
Incorrect example in documentation

### ISSUE THREAD

**Issue by rfleschenberg** [terraform-providers/terraform-provider-digitalocean#358]
Hi,

I am not sure if this is the right repository to file this issue. If not, can you point me at the right one?

Terraform Version: Terraform v0.12.19

When trying to follow the example at
https://www.terraform.io/docs/providers/do/r/kubernetes_cluster.html#kubernetes-terraform-provider-example, I run into this error:

```
Error: Failed to load config (/home/rene/.kube/config; default context): invalid configuration: no configuration has been provided

  on taurus.tf line 13, in provider "kubernetes":
  13: provider "kubernetes" {
```

Adding `load_config_file = false` seem to fix this for me:

```yaml
provider "kubernetes" {
  load_config_file = false
  host  = digitalocean_kubernetes_cluster.taurus-test.endpoint
  token = digitalocean_kubernetes_cluster.taurus-test.kube_config[0].token
  cluster_ca_certificate = base64decode(
    digitalocean_kubernetes_cluster.taurus-test.kube_config[0].cluster_ca_certificate
  )
}
```

I suppose this should be added to the example in the docs?

**Comment by andrewsomething** [comment]
Thanks for pointing this out. The docs for the Kubernetes provider do claim that:

> If you have both valid configuration in a config file and static configuration, the static one is used as override. i.e. any static field will override its counterpart loaded from the config.

https://www.terraform.io/docs/providers/kubernetes/index.html

Though it does seem to try to do some validation on the config file even if a static config exists which allows for failures even with a valid static config. Adding `load_config = false` seems like the right thing to do here.

**Comment by eddiezane** [comment]
@rfleschenberg did some testing and this is because `load_config_file` does default to `true`.

I agree that we should add that to the examples.

If you want to get your first PR here (which we would absolutely love) you'd just need to add the line to the blocks in https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/website/docs/r/kubernetes_cluster.html.markdown and https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/website/docs/d/kubernetes_cluster.html.md

**Comment by rfleschenberg** [comment]
@eddiezane Thanks for the guidance! I opened #359.

**Comment by eddiezane** [comment]
Thanks a ton for the PR! Closed via #359
